packages/dev/test/RelayHub.test.ts

```diff
@@ -162,6 +164,39 @@ contract('RelayHub', function ([paymasterOwner, relayOwner, relayManager, relayW
       })
     })
 
+    it('accounts with deposits can withdraw to multiple destinations', async function () {
```

Comment:
nitpic: instead of complex withdraw, could simply withdraw "1 wei" to a random address, and 2 wei to another random address. no need for pre/post balance diff...